tests/marketprice/messages/bch.py

```diff
@@ -0,0 +1,31 @@
+"""Schema definitions for `marketprice.messages.bch` namespace. Generated by avro2py v.0.0.1."""
```

Comment:
it strikes me that since you have extensive generated examples here, if you've gone to the trouble to inspect them, would it make sense to regression test against them? 

like, "load the ast representation from the previously generated code, assert it's equal to what currently gets generated"